frontend/src/pages/SignUpPage.js

```diff
@@ -1,46 +1,57 @@
 import React, { useCallback } from 'react';
-import { useHistory } from 'react-router-dom';
 import { gql, useMutation } from '@apollo/client';
 
 import { SignUpTemplate } from 'src/templates/SignUpTemplate';
-import { useAuth } from 'src/utils/auth';
 
 const SIGNUP_MUTATION = gql`
   mutation SignUp(
     $email: String!
-    $name: String!
     $password: String!
-    $userName: String!
+    $name: String!
+    $surname: String!
+    $height: Int!
+    $weight: Int!
+    $sex: String!
+    $birthdate: String!
   ) {
     signup(
       email: $email
-      name: $name
       password: $password
-      userName: $userName
-    ) {
-      user {
-        id
-        name
-        userName
-        profileImageUrl
-      }
-      token
-    }
+      name: $name
+      surname: $surname
+      height: $height
+      weight: $weight
+      sex: $sex
+      birthdate: $birthdate
+    )
   }
 `;
 
 export function SignUpPage() {
-  const auth = useAuth();
   const [signupRequest, signupRequestState] = useMutation(SIGNUP_MUTATION, {
-    onCompleted: ({ signup: { user, token } }) => {
-      auth.signin({ token, user });
+    onCompleted: ({ signup }) => {
```

Comment:
tady bys měl asi něco ukázat, ne? ne jen zalogovat že je všechno gucci

Reply:
Jasne to udelam jeste.

Reply:
Pridan callback, jen pak bude potreba to napojit na ten SuccessMessage, co jsme delali vcera s Honzou.